### PR TITLE
Updated Horizon Plugin charm with unit tests and changes in reactive handlers

### PR DESCRIPTION
**Pull request by sachin-trilio**
1. Added basic unit tests using tox
2. Removed config parameter for tvault version
3. Fetching tvault version from the appliance using get_version function
4. Updated config variable names, return values

### CODE REVIEW

File: juju-charms/charm-trilio-horizon-plugin/src/old [hunk 1]
```diff
@@ -0,0 +1,9 @@
+options:
```
Comment (abhijeetpatra) [inline juju-charms/charm-trilio-horizon-plugin/src/old]: Is this file needed to be there ?

Comment (sachin-trilio) [reply juju-charms/charm-trilio-horizon-plugin/src/old]: No. Removed. Thanks.
